src/components/Dashboard/Dashboard.tsx

```
@@ -0,0 +1,362 @@
+              <div className="lg:col-span-2 space-y-8">
+                <div className="bg-white dark:bg-gray-900 p-6 rounded-xl border border-gray-100 dark:border-gray-800 shadow-sm">
+                   <div className="flex items-center justify-between mb-6">
+                      <h3 className="font-bold flex items-center space-x-2"><TrendingUp className="w-5 h-5 text-green-500" /><span>Organization Activity</span></h3>
+                   </div>
+                   <ContributorActivity data={activityData} name="Organization Total" />
+                </div>
+                <div className="grid grid-cols-1 md:grid-cols-2 gap-4">
+                   <div 
+                        onClick={() => openContributorModal('Total Contributors', allContributors)}
+                        className="p-6 bg-gradient-to-br from-blue-500 to-indigo-600 rounded-2xl text-white shadow-lg shadow-blue-500/20 cursor-pointer transform hover:scale-[1.02] transition-transform"
+                   >
```

Biome (2.4.4)

[error] Enforce to have the onClick mouse event with the onKeyUp, the onKeyDown, or the onKeyPress keyboard event.

(lint/a11y/useKeyWithClickEvents)